clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -747,7 +752,7 @@ public void testDelayedReconciliationResultAppliedWhenTargetChangedWithMetadataU
         membershipManager.poll(time.milliseconds());
 
         assertEquals(Collections.emptySet(), membershipManager.topicsAwaitingReconciliation());
-        verify(subscriptionState).assignFromSubscribed(topicPartitions(topic2Assignment, topic2Metadata));
+        verify(subscriptionState).assignFromSubscribedAwaitingCallback(eq(topicPartitions(topic2Assignment, topic2Metadata)), anyCollection());
```

Comment:
similar to above comment, here the second param we know it should be the same set of partitions 1 and 2 for topic2 (exactly the same as the first param, because the assignment is expected to be [t2p1, t2p2], and the added partitions the same set, because the member did not own any of them)